wallet/store.go

```diff
@@ -156,7 +156,7 @@ func (k *WalletKey) decrypt(ciphertext []byte, nonce []byte) (plaintext []byte,
 	return
 }
 
-func (k *WalletKey) Open(file *os.File) (w *Wallet, err error) {
+func (k *WalletKey) Open(file *os.File, debugMode bool) (w *Wallet, err error) {
```

Comment:
`file` can just be an `io.Reader` (and below at the method `Export` it can just be an `io.Writer`).